packages/trip-form/src/ModeButton/index.js

```diff
@@ -36,7 +36,10 @@ const ModeButton = props => {
       </Styled.ModeButton.Button>
 
       {title && showTitle && (
-        <Styled.ModeButton.Title className={disabledClassName} title={title}>
+        <Styled.ModeButton.Title
+          className={`${activeClassName} ${disabledClassName}`}
```

Comment:
We should not use classes like this with styled-components. Instead, the styled component should provide dynamic values based on the props that are sent. So this should be rewritten as:

```css
<Styled.ModeButton.Title
  selected={selected}
  title={title}
>
```